main.js

```diff
@@ -0,0 +1,32 @@
+const navItems = document.querySelectorAll("nav ul li");
+console.log(navItems);
+
+for(let el of navItems){
+    if(el.textContent.includes("All Posts")){
+        el.remove();
+    }
+}
```

Comment:
```suggestion
let firstLi = document.querySelector("li")
firstLi.remove()
OR
document.querySelector("li").remove();
```
Since the "All Posts" link happens to be the first `<li>`, you can just write it like this. However, I do like that you were very thorough and used a for loop/if statement. This way the "All Posts" link will be removed whether it's the first `<li>` or not.